tests/basic/pool_function_test.go

```diff
@@ -533,5 +520,84 @@ var _ = Describe("{PoolExpandWhileResizeDiskInProgress}", func() {
 		})
 
 	})
+})
+
+var _ = Describe("{PoolExpandTestLimits}", func() {
+	BeforeEach(func() {
+		contexts = scheduleApps()
+	})
+
+	JustBeforeEach(func() {
+		poolIDToResize = pickPoolToResize()
+		log.Infof("Picked pool %s to resize", poolIDToResize)
+		poolToResize = getStoragePool(poolIDToResize)
+	})
+
+	JustAfterEach(func() {
+		AfterEachTest(contexts)
+	})
+
+	AfterEach(func() {
+		appsValidateAndDestroy(contexts)
+		EndTorpedoTest()
+	})
+
+	It("Initiate pool expansion (DMThin) to its limits (15 TiB) and beyond", func() {
+		Step("Expand pool to 15 TiB (max supported capacity for DMThin) with resize-disk type. ", func() {
+			var testrailID = 51292
+			// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/51292
+
+			StartTorpedoTest("PoolExpandTestWithin15TiBLimit",
```

Comment:
What's the implication of calling `StartTorpedoTest` within a `Step` block within an `It` block in terms of logs pushed to aetos?  @lsrinivas-pure

Reply:
@alicelyu-px here also if any failure in BeforeEach or JustBeforeEach will not report it to aetos

Reply:
Removed `Step` block as it was not necessary and just kept `It` block, like other existing tests.